gulp/tasks/lint.js

```diff
@@ -5,6 +5,7 @@ var gulp = require( 'gulp' );
 var gulpEslint = require( 'gulp-eslint' );
 var handleErrors = require( '../utils/handle-errors' );
 var minimist = require( 'minimist' );
+var through = require( 'through2' );
```

Comment:
ok, fixed in https://github.com/cfpb/cfgov-refresh/pull/3010/commits/16fdc07dd6f8dc11b32ad1e527109ebfa7bb75c8.